sources/engine/Xenko.Games/GameBase.cs

```diff
@@ -165,50 +140,27 @@ protected GameBase()
         #region Public Properties
 
         /// <summary>
-        /// Gets the current update time from the start of the game.
+        /// The current time from the start of the game, this one should be used for any logic running within the update loop.
         /// </summary>
-        /// <value>The current update time.</value>
-        public GameTime UpdateTime
-        {
-            get
-            {
-                return updateTime;
-            }
-        }
+        public GameTime UpdateTime { get; }
 
         /// <summary>
-        /// Gets the current draw time from the start of the game.
+        /// The current time from the start of the game, this one should be used for any logic running within the draw loop.
```

Comment:
Put the second part of the sentence either in a second sentence or under `<remarks>`.

Reply:
See conversation above.